### PR TITLE
Legacy configuration loading for zoom and center

### LINKED ISSUE #24: Add initial support to remote configuration loading
Allow the application to load zoom and center from a remote configuration.

### LINKED ISSUE #25: Fix webpack configuration for tests to load jsx file
Actually the require don't include jsx if running with karma. The karma webpack confiuguration should allow to do that.

### PR DESCRIPTION
**Pull request by offtherailz**
This first implementation loads legacy configuration file and create a configuration object that can be a mock for the new future configuration. 

* The tests for this task caused errors because of some nested requires to jsx. This has been solved. close #25
* Remote configuration loading. close #24

### CODE REVIEW

File: web/client/api/MapConfigDAO.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ * LICENSE file in the root directory of this source tree.
+ */
+
+
+var axios = require('axios');
+
+/**
+ * API for local config
+ */
+var Api = {
```
Comment (mbarto) [inline web/client/api/MapConfigDAO.js]: Do we need this? Can't we do directly axios.get(url) where needed?

Comment (offtherailz) [inline web/client/api/MapConfigDAO.js]: It is a skeleton for configuration DAO. Can be used for remote configuration managing:
* errors
* headers
* authentication

File: web/client/components/Map/__tests__/config-test.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+/**
```
Comment (mbarto) [inline web/client/components/Map/__tests__/config-test.jsx]: We should rename the file to MapViewController-test.jsx

File: web/client/utils/ConfigUtils.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    convertFromLegacy: function(config) {
+        var mapConfig = config.map;
+        var layers = mapConfig.layers;
+        var center = mapConfig.center;
+        var zoom = mapConfig.zoom;
+        var maxExtent = mapConfig.maxExtent;
+        var epsg900913 = new Proj4js.Proj('EPSG:900913');
+        var epsg4326 = new Proj4js.Proj('EPSG:4326');
+        var xy = new Proj4js.Point(center);
+        Proj4js.transform(epsg900913, epsg4326, xy);
```
Comment (mbarto) [inline web/client/utils/ConfigUtils.js]: Let's check the config srs before doing the projection

**Pull request merged**